### ISSUE TITLE
Prior deleted deploy showing up instead of newly deployed site

### ISSUE THREAD

**Issue by badrhombus** [aws-amplify/amplify-console#688]
**App Id**: d2lwend04neamk
**Region**: us-east-1
**Step**: 2  Troubleshoot Domain - 
**Status**: Domain activation -X- Failed
**Error Message**: One or more of the CNAMEs you provided are already associated with a different resource.

**Issue/question**
I Used Route 53 to set up A Record, CNAME domain and CNAME www records, in addition I created a test.myDomain.com CNAME record.

They all point to the same 12345678901234.cloudfront.net. CNAME but the www.myDomain.com and the top domain (myDomain.com) both go to an old version from different app id, however test.myDomain fails with a 403 page.

I checked the DNS and they are all delivering the same set of IPs

**Additional information**
FYI: new version is correctly deployed and can be gotten from the *********.amplifyapp.com url

test.myDdomain.com 
403 ERROR
The request could not be satisfied.
...

**Comment by badrhombus** [comment]
So I may be getting **CNAMEAlreadyExistsException** with a prior, now deleted version of the app. How can I remove the old content and delete the CNAME association to the original CloudFront.net distribution? I need to delete CNAME association and replace it with the new one.

**Comment by badrhombus** [comment]
I found the prior App Id **d1al6j15fy5a30** but do not know where it is being maintained other than the cloudfront.net cache, Is it possible to purge it entirely so that I can create a new CNAME reference without conflict?

**Comment by badrhombus** [comment]
My issue was cause by a forgotten Amplify Console app generated in another region.
The advise by @Athena96 was to identify the App ID and Zone of the prior apps that I made.
It turns out, that although I thought I had deleted all prior apps and resources. One remained in another Region (which was selected inadvertently). 

The puzzler was that the creating DNS entries for the new region with new App ID caused the old version created in the undeleted prior (errant) region to be referenced just fine. Making the new App in the new region orphaned from the custom Domain.

Thanks for the Amplify CI and Hosting with CloudFront - this is great stuff
